media/js/plugincheck/lib/plugincheck.js

```diff
@@ -14,17 +14,19 @@
      */
     function setJavaVersion() {
         var type;
-        var installedVersion;
+        var installedVersion = '0.0.0'; // fallback is better than returning undefined?
```

Comment:
You are quite right, however the page then lists the plugin as up-to-date. We could go that route, but alternatively `0.0.0` results in the plugin being listed as out-of-date. Would it be better to err on the side of caution just in case the plugin is vulnerable?